README.md

```diff
@@ -71,19 +71,23 @@ devtools::install_github("tensorflow/io", subdir = "R-package")
 
 ### Python
 
-The TensorFlow I/O package (`tensorflow-io`) could be built from source:
+For Python development, a reference Dockerfile [here](dev/Dockerfile) can be
+used to build the TensorFlow I/O package (`tensorflow-io`) from source:
 ```sh
-$ docker run -it -v ${PWD}:/working_dir -w /working_dir tensorflow/tensorflow:custom-op
-$ # In docker
-$ curl -OL https://github.com/bazelbuild/bazel/releases/download/0.20.0/bazel-0.20.0-installer-linux-x86_64.sh
-$ chmod +x bazel-0.20.0-installer-linux-x86_64.sh
-$ ./bazel-0.20.0-installer-linux-x86_64.sh
+$ # Build and run the Docker image
+$ docker build -f dev/Dockerfile -t tfio-dev .
+$ docker run -it -rm -v ${PWD}:/v -w /v tfio-dev
+$ # In Docker, configure will install TensorFlow or use existing install
 $ ./configure.sh
-$ bazel build build_pip_pkg
-$ bazel-bin/build_pip_pkg artifacts
+$ # Build TensorFlow I/O C++
+$ bazel build -s --verbose_failures --spawn_strategy standalone //tensorflow_io/...
+$ # Run tests with PyTest, note: some tests might require starting services to run
```

Comment:
I actually had some trouble running ignite and kafka tests, I tried running the provided containers but couldn't connect to them. Do I need to forward some ports? Just wondering how you all run these..

Reply:
@BryanCutler You can pass `docker run -it -rm --net=host -v ${PWD}:/v -w /v tfio-dev` to get access to host port from within docker container.

Reply:
Ah, thanks I forgot the `--net` option. I'll update with the info